lib/senders/email.js

```diff
@@ -267,50 +267,51 @@ module.exports = function (log, config) {
       optionalHeader('X-Uid', message.uid)
     )
 
-    var emailConfig = {
-      sender: this.sender,
-      from: this.sender,
-      to: message.email,
-      subject: localized.subject,
-      text: localized.text,
-      html: localized.html,
-      xMailer: false,
-      headers
```

Comment:
Some of the code in this function was working with properties of this `emailConfig` object and other bits were using local aliases. That made for ambiguous reading, for instance is `emailConfig.headers` the same as `headers` or are they different? Which one am I supposed to use? etc

Because of that, I opted to move `emailConfig` down to the point immediately before it's passed in to `mailer.sendMail` and change the rest of the code to always use the local alias. I think/hope that will avoid confusion in the future.